Configuration/TSconfig/ContentElementWizard.txt

```diff
@@ -11,3 +11,22 @@ mod.wizards.newContentElement.wizardItems.plugins {
         }
     }
 }
+
+#Marker for the Configuration/TCA/tx_seminars_seminars.php
```

Comment:
Please let's describe this more in terms of what these lines do, not to which files they relate.